app.py

```diff
@@ -15,6 +15,10 @@
 
 from kwek.database import db
 
+import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
```

Comment:
Let's make this configurable, i.e. add a `DISABLE_WARNINGS` configuration variable  on `api/__init__.py`.